modules/fundamental/src/sbom/model/raw_sql.rs

```diff
@@ -0,0 +1,129 @@
+/// This function returns SQL subquery that filters the context_cpe_id
+/// based on the given sbom_id. It checks if the context_cpe_id is null
+/// or if it is in the list of CPEs that are related to the packages
+/// that describes the SBOM. The additional logic allows us to find
+/// superset of generalized CPEs that don't include subfields like edition
+/// and find "zstream" releases based on the major version.
+pub fn context_cpe_filter_sql() -> &'static str {
```

Comment:
I guess (not sure) you can make those `fn`s `const`. So that they would be run a compile time.

Reply:
Cool. Let me give it a try